### ISSUE TITLE
Improve the output PDF names

### ISSUE THREAD

**Issue by bountonw** [bountonw/translate#98]
Part of PR #89 .

From: PP01_th.pdf
To: [book code][chapter ##]_[language code]_[status]_[timestamp].pdf

**Comment by bountonw** [comment]
@mattleff Is this done?

**Comment by mattleff** [comment]
@bountonw No, it's not. At least not per the naming structure in the description. Right now we are mirroring the naming structure of the .md file in the generated .pdf. If that's ok then we can close this.
